lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -31,6 +31,7 @@ type StudentsTableRow = {
   annotations: number;
   replies: number;
   current_grade?: number;
+  last_grade?: number | null;
```

Comment:
Perhaps rename to `last_submitted_grade` to make it clear what point in time "last" refers to, or add JSDoc comments which explain this.